spec/addressable/idna_spec.rb

```diff
@@ -38,6 +38,12 @@
     )).to eq("www.xn--8ws00zhy3a.com")
   end
 
+  it "also accepts unicode strings encoded as ascii-8bit" do
+    expect(Addressable::IDNA.to_ascii(
+      "www.詹姆斯.com".b
+    )).to eq("www.xn--8ws00zhy3a.com")
+  end
+
```

Comment:
The ruby normalize function requires a proper Unicode string which was not the case before, so even though this was not the case anywhere, I though I would add a spec to make sure it still works even if the user (or other code) passes a wrongly encoded string (`ASCII-8BIT` in this example thanks to `.b`)